src/app/lib/utilities/preprocessor/rules/filterContentType.js

```diff
@@ -18,6 +18,15 @@ const filterUnknownContentTypes = data => {
       newGroup.items = group.items.filter(item => {
         const itemType = item.assetTypeCode || item.cpsType;
         const validItemType = whitelist.includes(itemType);
+
+        if (group.type === 'useful-links') {
```

Comment:
I found the conditions inside this filter a little hard to understand looking at it for the first time - would something like this be a little easier to read?

```
const itemType = item.assetTypeCode || item.cpsType;
const isValidItemType = whitelist.includes(itemType);
const isValidContentType = itemType !== 'PRO' || contentTypes.includes(item.contentType);
const isValidGroupType = group.type !== 'useful-links' || itemType === 'PRO';
return isValidItemType && isValidContentType && isValidGroupType;
```